### ISSUE TITLE
Cannot resolve a promise with a can.List using can/list/promise

### ISSUE THREAD

**Issue by dylanrtt** [canjs/canjs#2382]
The following never resolves when using the can/list/promise plugin.

```
Promise.resolve(new can.List());
```

I assume that this is because the list is perceived as a promise, but I don't see a good reason for the list to be promise-like if the list is not instantiated with a params object.

**Comment by justinbmeyer** [comment]
If we bring back `can-list-promise` in 3.0, we should fix it there and possibly back-port to 2.3.

**Comment by justinbmeyer** [comment]
I'm going to close.  If this gets a few people adding +1s, we can make the `can-list-promise` plugin.  However, a workaround is to simply use promises in can-stache:

```
{{#if promise.isResolved}}
  {{#each promise.value}}
  ...
```
